### PR TITLE
Feature- fixing pattern design

### PR DESCRIPTION
**Pull request by stospina**
Please code review

### CODE REVIEW

File: MACHINE-LEARNING-SERVICE/src/helpers/handler.file.js [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+This software is the confidential and proprietary information of
+Jalasoft Confidential Information You shall not
+disclose such Confidential Information and shall use it only in
+accordance with the terms of the license agreement you entered into
+with Jalasoft.
+*/
+
+const Decompress = require('./decompress.helper');
+const path = require('path');
+
```
Comment (florpadilla) [inline MACHINE-LEARNING-SERVICE/src/helpers/handler.file.js:14]: please add a comment to this class

**Review by florpadilla — COMMENTED** [review]
It looks good. Just add a comment at handler.file.js

File: MACHINE-LEARNING-SERVICE/src/helpers/handler.file.js (hunk 1 above)
Comment (stospina) [reply MACHINE-LEARNING-SERVICE/src/helpers/handler.file.js:14]: Done... thank you

Comment (florpadilla) [reply MACHINE-LEARNING-SERVICE/src/helpers/handler.file.js:14]: great!

**Review by florpadilla — APPROVED** [review]
Everything is ok!

**Pull request merged**